src/main/java/core/basesyntax/service/impl/ReturnAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class ReturnAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
+        Storage.addFruit(fruit, quantity);
```

Comment:
The direct call to 'Storage.addFruit' makes 'ReturnAction' tightly coupled with 'Storage', which violates the SOLID principles. Consider adding a service layer to interact with the storage.